### PR TITLE
Remove "Skeleton" Loading Screen

### PR DESCRIPTION
**Pull request by kkosiorowska**
Closes https://github.com/thesis/acre/issues/811

The "Skeleton" loading screen from both Deposit and Withdraw flows is no longer needed. So let's remove this screen from the staking retry step as well.

https://github.com/user-attachments/assets/180bc4ce-b03f-47b1-b2d1-647cd20f4ca2

### CODE REVIEW

File: dapp/src/components/TransactionModal/ActiveStakingStep/StakingErrorModal/RetryModal.tsx [hunk 1]
```diff
@@ -29,7 +31,13 @@ const getCounterData = (minutes: string, seconds: string) => {
   return { label, progressPercent }
 }
 
-export default function RetryModal({ retry }: { retry: () => void }) {
+export default function RetryModal({
+  isLoading,
+  retry,
+}: {
+  isLoading: boolean
```
Comment (kpyszkowski) [inline dapp/src/components/TransactionModal/ActiveStakingStep/StakingErrorModal/RetryModal.tsx:38]: NB:
Let's make the type definition explicit for consistency. 
Like `type RetryModalProps = { ... }`

**Review by kpyszkowski — CHANGES_REQUESTED** [review]
Left non-blocking comment:

File: dapp/src/components/TransactionModal/ActiveStakingStep/StakingErrorModal/RetryModal.tsx (hunk 1 above)
Comment (kkosiorowska) [reply dapp/src/components/TransactionModal/ActiveStakingStep/StakingErrorModal/RetryModal.tsx]: https://github.com/thesis/acre/pull/828/commits/ebc043b071a89ca671b95455d652755b850f1cda

**Review by kpyszkowski — APPROVED** [review]
LGTM 🚀

**Pull request merged**